tests/components/application-connector/test/compass-runtime-agent/suite_test.go

```diff
@@ -44,7 +56,41 @@ func (gs *CompassRuntimeAgentSuite) SetupSuite() {
 
 	applicationGetter := gs.applicationsClientSet.ApplicationconnectorV1alpha1().Applications()
 	gs.appComparator, err = applications.NewComparator(gs.Require(), secretComparator, applicationGetter, "", "")
-	gs.Require().Nil(err)
+	//gs.T().Log("Attempt to unregister application in compass")
```

Comment:
It think we can remove the commented code

Reply:
Removed